R/visr_plot.R

```diff
@@ -101,13 +109,13 @@ plot.survfit <- function(
       log = "log(Survival probability)",
       event = "Failure probability",
       cloglog = "log(-log(Survival probability))",
-      pct = "Survival probability (%)",
-      logpct = "log(Survival probability (%))",
+      pct = "Survival percentage (%)",
```

Comment:
Not sure if this is the correct term.

Reply:
Agree. Survival percentage makes more sense. Also we can then drop the %.

Reply:
Same for log surv percentage

Reply:
Did you mean Percentage survival makes more sense? (survival percentage is what it is saying now)

Reply:
Percentage survival.